### DISCUSSION TITLE
Dark theme?

### DISCUSSION THREAD

**Discussion by noncom** [rabbibotton/clog#387] [General]
Hello!

I can't seem to figure out how to set a fully dark theme in CLOG.
Am I missing the setting somewhere, or is it not implemented?
I mean a 100% dark themed UI, not just the code editor.

**Comment by rabbibotton** [comment]
In preferences you will see settings you can adjust color wise etc. menu
bars, etc. I once had a sample around.

**Reply by noncom** [reply]
Just got back to it, and I tried finding preferences, but there's none I can find.
When I start CLOG, it opens the Builder in the browser, and there are some menus, but there's no preferences, options or settings of any kind.
I'm sorry if I'm missing something obvious, but where do I find preferences? :)

**Comment by rabbibotton** [comment]
Are you using UltraLisp or clog 1 from QuickLisp

**Reply by noncom** [reply]
Hmm, I think it appeared to be I was using clog 1 from QuickLisp since I skipped right to the Quick Start instruction in the readme, which told me to:
```
sbcl --eval "(ql:quickload :clog/tools)" --eval "(clog-tools:clog-builder)"
```
and which I did, but the UltraLisp instruction was before that, in the previous section.

I now stepped back, added UltraLisp, and tried loading clog again.
So yeah, now it loaded what looks like a much more advanced environment :)

Thank you! I will experiment with it!
